src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Label/BitLabelTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Bunit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bit.Client.Web.BlazorUI.Tests.Label
+{
+    [TestClass]
+    public class BitLabelTests: BunitTestContext
+    {
+        [TestMethod]
+        public void IsRequired_HasRequiredClass_True()
```

Comment:
Could you please test data for this, so we can test both required and optional labels at the same time?